EIPS/eip-1895.md

```diff
@@ -0,0 +1,166 @@
+---
+eip: 1895
+title: Support for an Elliptic Curve Cycles
```

Comment:
I'm thinking you'd want the title to either say 

- *Support for Elliptic Curve Cycles* 
or
- *Support for an Elliptic Curve Cycle*

not 
- *...an Elliptic Curve Cycles*